}

#forum .container {
  background: #a1a1a1;
  color: #fff;
}

#forum .container a {
  color: #000;
  font-weight: bold;
}

#forum div.indent {
  margin-left: 2px;
}

.forum-starter-picture {
  float: left;
  padding: 4px;
  border: 1px solid #d9d9d9;
  margin: 5px 7px 5px 0;
}

#forum th {
  background: #434343;
  text-shadow: none;
  color: #fff;
}

#forum th a {
  color: #fff;
}

#forum td.forum .icon, #forum .icon div {
  background: url("../images/no-new-posts.png") no-repeat;
  width: 35px;
  height: 35px;
}

#forum .name a {
  font-size: 16px;
}
<div id="container">
        <div class="content-sidebar-wrap">
    <div id="content">
      <div id="breadcrumbs"><h2 class="element-invisible">You are here</h2><nav class="breadcrumb"><a href="/raghava/">Home</a> » <a href="/raghava/webservices" class="active-trail">Web Services</a> » Web Services (A to Z)</nav></div>      <section id="post-content" role="main">
                                <h1 class="page-title">Web Services (A to Z)</h1>                                        <div class="region region-content">
  <div id="block-system-main" class="block block-system">
        <div class="content">
                          <span property="dc:title" content="Web Services (A to Z)" class="rdf-meta element-hidden"></span><span property="sioc:num_replies" content="0" datatype="xsd:integer" class="rdf-meta element-hidden"></span>            <div class="content node-page">
    <div class="field field-name-body field-type-text-with-summary field-label-hidden"><div class="field-items"><div class="field-item even" property="content:encoded"><p>Web Server/Software Developed at Raghava's Group</p>
<ol><li><a href="http://webs.iiitd.edu.in/raghava/abcpred/" target="_blank">ABCpred</a>: Mapping of B-cell epitope(s) in an antigen sequence, using artificial neural network.</li>
<li><a href="http://crdd.osdd.net/raghava/abmpred/" target="_blank">abmpred</a>:Prediction of AntiBacterial Compounds against MurA Enzyme.</li>
<li><a href="http://crdd.osdd.net/raghava/absource/" target="_blank">absource</a>:All possible informations regarding the antibodies.</li>
<li><a href="http://webs.iiitd.edu.in/resources/webs.iiitd.edu.in/raghava/progs/abaffi/README.html" target="_BLANK">Ab-Affi:&nbsp;</a>Computer program for calculating affinity of a monoclonal antibody using non-competitive ELISA.</li>
<li><a href="http://crdd.osdd.net/raghava/adpint/" target="_blank">ADPint</a>:Predicion of ADP interacting residue.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ahtpin/" target="_blank">AHTpin</a>: Designing and virtual screening of antihypertensive peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/algpred/" target="_blank">ALGpred</a>: Prediction of allergenic proteins and mapping of IgE epitopes in antigens.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/alphapred/" target="_blank">ALPHApred</a>: A neural network based method for predicting alpha-turn in a protein.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/antiangiopred/" target="_blank">AntiAngiopred</a>: A Server for Prediction of Anti-Angiogenic Peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/antibp/" target="_blank">AntiBP</a>: Mapping of antibacterial peptides in a protein sequence.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/antibp2/" target="_blank">AntiBP2</a>: Mapping of antibacterial peptides in a protein sequence.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/anticp/" target="_blank">AntiCP</a>: Prediction and design of anticancer peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/antifp/" target="_blank">AntiFP</a>: A Prediction server for Antifungal Peptide.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/antigendb/">AntigenDB:</a>&nbsp;an immunoinformatics database of pathogen antigens.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/antitbpdb/" target="_blank">AntiTbPdb</a>: A database of anti-tubercular peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/apocand/" target="_blank">ApoCanD</a>:&nbsp; Database of Human Apoptotic Proteins in the context of cancer.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/apssp/" target="_blank">APSSP</a>:&nbsp; Advanced Protein Secondary Structure Prediction Server.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/apssp2/" target="_blank">APSSP2</a>: Prediction of secondary structure of proteins from their amino acid sequence.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ar_nhpred/" target="_blank">Ar_NHpred</a>: Identification of aromatic-backbone NH interaction in protein residues.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/atpint/" target="_blank">ATPint</a>: Identification of ATP binding sites in ATP-binding proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/bcepred/" target="_blank">BCEpred</a>: Prediction of linear B-cell epitopes, using Physico-chemical properties.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/bcipep/" target="_blank">BCIpep</a>: Collection &amp; compilation of B-cell epitopes from literature.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/betatpred/" target="_blank">BetaTPred</a>: Statistical-based method for predicting Beta Turns in a protein.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/betatpred2/" target="_blank">BetaTPred2</a>: Statistical-based method for predicting Beta Turns in a protein.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/betatpred3/">BetaTPred3</a>: Prediction of&nbsp;beta turns and their types.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/betaturns/">BetaTurns</a>:&nbsp;Prediction of&nbsp;beta turn types.</li>
<li><a href="http://www.imtech.res.in/raghava/bhairpred/" target="new">BhairPred:&nbsp;</a>Prediction of beta hairpins in proteins using ANN and SVM techniques.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/biadb/">BIAdb</a>:&nbsp;A Database for Benzylisoquinoline Alkaloids.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/biotherapi/">BioTherapi</a>:&nbsp;Bioinformatics for Therapeutic Peptides and Proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/bteval/" target="_blank">BTEVAL</a>: Evaluation of beta-turns prediction methods.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/btxpred/">BTXpred</a>: A server for predicting bacterial toxins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancercsp/">CancerCSP</a>: Gene expression-based biomarkers for discriminating early and late stage of clear cell renal cancer.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancerdp/">CancerDP</a>: Prioritization of anticancer drugs against a cancer using genomic features of cancer cells.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancerdr/">CancerDR</a>:&nbsp;Cancer drug resistance database.</li>
<li><a href="http://crdd.osdd.net/raghava/canceres" target="_BLANK">CanceRES:&nbsp;</a>Cancer Informatics Portal.</li>
<li><a href="http://crdd.osdd.net/raghava/cancerin/" target="_blank">CancerIN</a>:A web server for predicting anticancer activity of molecules.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancerpdf/">CancerPDF</a>: Database of cancer-associated peptides of&nbsp;human bodyfluids.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancerppd/">CancerPPD</a>:&nbsp;&nbsp;Database of anticancer peptides&nbsp;and proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancer_pred/">CancerPred</a>: Prediction of&nbsp;the cancerlectins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancertope/">Cancertope</a>:&nbsp;A Platform for Designing Genome-Based Personalized Immunotherapy or Vaccine against Cancer.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cancerubm/">CancerUBM</a>: Web bench for analysis of&nbsp;proteomics data and&nbsp;prediction of&nbsp;cancer status of a urine samples.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cbtope/">CBTOPE</a>: Conformational B-cell Epitope prediction.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ccdb/">CCDB</a>: Database of genes involved in Cervix Cancer.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ccpdb/">ccPDB</a>: Compilation and Creation of datasets from PDB.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cellppd/" target="_BLANK">CellPPD:&nbsp;</a>Computer-aided Designing of efficient cell penetrating peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/chemopred/">ChemoPred</a>: A server to predict chemokines and their receptor</li>
<li><a href="http://webs.iiitd.edu.in/raghava/chloropred/" target="new">ChloroPred:&nbsp;</a>Prediction of chloroplast proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/chpredict/" target="new">CHpredicts:&nbsp;</a>Prediction of CH-O, CH-PI interactions in backbone residues of a protein&nbsp;</li>
<li><a href="http://crdd.osdd.net/raghava/codon/" target="_blank">codon</a>:Plot Graph for Codon Composition (from standard codon usage table) Vs Codons.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/copid/" target="new">COPid:&nbsp;</a>Composition based identification and classification of proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cppsite1/">CPPsite:</a>&nbsp;A web page for cell penetrating peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cppsite" target="_BLANK">CPPsite 2.0:&nbsp;&nbsp;</a>A database of experimentally validated cell penetrating Peptides.</li>
<li><a href="http://webs.iiitd.edu.in/crdd/">CRDD:</a>&nbsp;Computational resources for drug discovery.</li>
<li><a href="http://crdd.osdd.net/raghava/crip" target="_BLANK">CRIP:&nbsp;</a>Computational Resources for molecular Interaction Prediction</li>
<li><a href="http://crdd.osdd.net/raghava/csirdb/" target="_blank">csirdb</a>:Identification of Cancer-specific siRNAs for personalized therapy.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ctlpred" target="_BLANK">CTLPred:&nbsp;</a>A direct method for prediction of CTL epitopes.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/cytopred/" target="new">CytoPred:&nbsp;</a>It is a webserver for prediction and classification of cytokines.</li>
<li><a href="http://bioinfo.icgeb.res.in/cyclinpred/" target="new">CyclinPred:&nbsp;</a>CyclinPred is a SVM based prediction method to identify novel cyclins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/dadpred/" target="_blank">DADpred</a>: Prediction of drugs against aspartate semialdehyde dehydrogenase of DAP pathway.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/dampro/" target="_blank">DAMpro</a>: Disease Associated Mutations in Proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/dbem/#">dbEM</a>: A Database of Epigenetic Modifiers</li>
<li><a href="http://webs.iiitd.edu.in/raghava/desirm/" target="new">desiRam:&nbsp;</a>Designing of highly efficient siRNA with minimum mutation approach</li>
<li><a href="http://webs.iiitd.edu.in/raghava/dipcell/">DiPCell:&nbsp;</a>Designing of inhibitors against pancreatic cancer cell lines.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/dnabinder/">DNAbinder:&nbsp;</a>A webserver for predicting DNA-binding proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/dnaint/" target="_blank">DNAint</a>: Prediction of DNA Interacting Residues.</li>
<li><a href="http://webs.iiitd.edu.in/resources/webs.iiitd.edu.in/raghava/progs/dnaopt/README.html" target="_BLANK">DNAOPT:&nbsp;</a>Computer program for&nbsp;optimizing the conditions of gel electrophoresis and SDS-PAGE.</li>
<li><a href="http://webs.iiitd.edu.in/resources/webs.iiitd.edu.in/raghava/progs/dnasize/" target="_BLANK">DNASIZE:&nbsp;</a>Computer program for computing the size of DNA fragments from gel data using a graphical method.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/dprot/">DPROT:</a>SVM-based method for predicting of disordered proteins.</li>
<li><a href="http://webs.iiitd.edu.in/oscadd/drugmint/">DrugMint</a>: A web server for identification of drug like molecules.</li>
<li><a href="http://crdd.osdd.net/oscadd/ebola/">EbolaVCR</a>:&nbsp;A web-based resource for designing therapeutics against Ebola Virus.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ecdb/#">ECDB</a>: Endometrial cancer gene database.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ecgpred/" target="new">ECGPred:&nbsp;</a>Analsis of expresion data and correlation between gene expression and nucleotides composition of genes.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/egfrindb/">EGFRindb:</a>&nbsp;A database of EGF(Epidermal Growth Factor) receptors &nbsp;(anticaner database).</li>
<li><a href="http://webs.iiitd.edu.in/oscadd/egfrpred/">EGFRpred</a>: A&nbsp;web service for the predicting and designing of inhibitors against EGFR (anticancer drugs).</li>
<li><a href="http://webs.iiitd.edu.in/raghava/egpred/" target="new">EGPred:&nbsp;</a>Prediction of gene (protein coding regions) in eukaryote genomes that includes introns/exons.</li>
<li><a href="http://webs.iiitd.edu.in/resources/webs.iiitd.edu.in/raghava/progs/elisaeq/README.html" target="_BLANK">ELISAeq:&nbsp;</a>Computer program for determining the concentration of&nbsp;antibody/antigen from ELISA data.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/eslpred/" target="new">ESLpred:&nbsp;</a>Subcellular localization of the eukaryotic proteins using dipeptide compostion and PSI-BLAST.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/eslpred2/" target="new">ESLpred2:&nbsp;</a>Advanced method for subcellular localization of eukaryotic proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/fadpred/" target="_blank">FADpred</a>&nbsp;: A webserver for the prediction of FAD interacting residues.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/fisipred/" target="_blank">fisipred</a>: Phi-Psi angle prediction using average angle prediction technique.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ftg" target="new">FTG:&nbsp;</a>Locating probable protein coding region in nucleotide sequence using FFT based algorithm.</li>
<li><a href="http://crdd.osdd.net/raghava/ftgpred/" target="_blank">ftgpred</a>:Gene Identification using Fourier Transformation.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gammapred/" target="new">GammaPred:&nbsp;</a>Identification of gamma-turn containing residues in a given protein sequence.</li>
<li><a href="http://www.imtech.res.in/genesis/">GENESIS</a><a href="http://webs.iiitd.edu.in/genesis/">:</a>&nbsp;It is a mega project initiated for establishing CSIR in the field of knowledge based discovery.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gdoq/">GDoQ:&nbsp;</a>Prediction of GLMU inhibitors (anti-tubeculosis inhibitors) using QSAR and docking apprach.</li>
<li><a href="http://crdd.osdd.net/raghava/genebench/" target="_blank">genebench</a>:A suite of tools and datasets for benchmarking gene finders.</li>
<li><a href="http://crdd.osdd.net/raghava/genomeabc/" target="_blank">genomeabc</a>:GenomeABC is a web server for evaluating the performance of genome assemblers.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/glycoep/" target="new">GlycoEP:&nbsp;</a>Prediction of C-, N- and O-glycosylation site in eukaryotic proteins.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/glycopp/" target="new">GlycoPP:&nbsp;</a>Prediction of potential N-and O-glycosites in prokaryotic proteins.</li>
<li><a href="http://webs.iiitd.edu.in/resources/webs.iiitd.edu.in/raghava/progs/gmap/README.html" target="_BLANK">GMAP:&nbsp;</a>Computer program&nbsp;for mapping potential restriction endonuclease&nbsp;sites in non-ambiguous DNA.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gmpesi/" target="_blank">GMPEsi</a>: Generalized Model for Predicting Efficacy of siRNAs.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gpcrpred/" target="new">GPCRpred:&nbsp;</a>Prediction of families and superfamilies of G-protein coupled receptors (GPCR).</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gpcrsclass/" target="new">GPCRsclass:&nbsp;</a>This webserver predicts amine type of G-protein coupled receptors</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gpsr/">GPSR:</a>&nbsp;A resource for genomocs, proteomics and system biology</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gstpred/" target="new">GSTpred:&nbsp;</a>SVM-based method for predicting Glutathione S-transferase protein.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/gtpbinder/" target="new">GTPbinder:&nbsp;</a>Identification of GTP binding residue in protein sequences.&nbsp;</li>
<li><a href="http://crdd.osdd.net/raghava/gwblast/" target="_blank">GWBLAST</a>:Genome-wide sequence similarity search using BLAST.</li>
<li><a href="http://crdd.osdd.net/raghava/gwfasta/" target="_blank">GWFASTA</a>: Genome-wide sequence similarity search using FASTA.</li>
<li><a href="http://crdd.osdd.net/raghava/gdpbio" target="_BLANK">GDPbio:&nbsp;</a>Genome based prediction of Diseases and Personal Medicine</li>
<li><a href="http://webs.iiitd.edu.in/raghava/haptendb" target="_BLANK">HaptenDB:&nbsp;</a>A database of haptens, provide comprehensive information about the hapten molecule.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hemolytik" target="_BLANK">HemolytiK:&nbsp;</a>A repository of experimentally validated hemolytic and non-hemolytic peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hemopi" target="_BLANK">HemoPI:&nbsp;</a>Prediction and virtual screening of hemolytic peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/herceptinr" target="_BLANK">HerceptinR:&nbsp;</a>Information about assays performend to test sensitivity/resistance of Herceptin Antibodies.</li>
<li><a href="http://crdd.osdd.net/raghava/hivbio" target="_BLANK">HIVbio:&nbsp;</a>HIV related bioinformatics resources.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hivcopred/" target="new">HIVcoPRED:&nbsp;</a>Predicting coreceptor used by HIV-1 from Its V3 loop amino acid sequence.</li>
<li><a href="http://webs.iiitd.edu.in/oscadd/hivfin/">HIVfin</a>: Prediction of Fusion Inhibitors against HIV (drugs against HIV)</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hivint/" target="_blank">HIVint</a>: Prediction of&nbsp;protein-protein interaction between HIV and Human proteins.</li>
<li><a href="http://crdd.osdd.net/raghava/hivsir" target="_BLANK">HIVsir:&nbsp;</a>A manually curated database of anti-HIV siRNAs.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hladr4pred/" target="_BLANK">HLA-DR4Pred:&nbsp;</a>Identification of HLA-DRB1*0401(MHC class II alleles) binding peptides.</li>
<li><a href="http://crdd.osdd.net/raghava/hlapred/" target="_blank">HLApred</a>:Identification and prediction of HLA class I and class II Binders.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hlp/" target="_BLANK">HLP:&nbsp;</a>Designing of stable antibacterial peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hmrbase" target="_BLANK">HMRbase:&nbsp;</a>A manually curated database of hormones and their Receptors.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/hslpred" target="new">HSLpred:&nbsp;</a>Prediction of subcellular localization of human proteins with high accuracy</li>
<li><a href="http://webs.iiitd.edu.in/raghava/humcfs/">HumCFS</a>: A database of fragile sites in human chromosome.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/icaars/">icaars</a>: Identification &amp; Classification of Aminoacyl tRNA Synthetases.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ifnepitope/">IFNepitope</a>: A server for predicting and designing interferon-gamma inducing epitopes.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/igpred/">IgPred</a>: Prediction of antibody specific B-cell epitope.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/il10pred/">IL10pred:</a>&nbsp;Prediction of Interleukin-10 inducing peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/il4pred/">IL4pred:</a>&nbsp;In silico platform for designing and disovering of Interleukin-4 inducing peptides.</li>
<li><a href="http://webs.iiitd.edu.in/resources/webs.iiitd.edu.in/raghava/progs/il4ifng/README.html" target="_BLANK">IL4IFNG:&nbsp;</a>Computer program for computing of murine interleukin-4 and interferon-gamma.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/imrna/">imRNA:</a>&nbsp;Prediction of Immunomodulatory RNAs.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/kidoq/">KiDoQ:</a>&nbsp;Prediction of inhibition against dihydrodipicolinate synthase of M.&nbsp;<em>Tuberculosis</em>.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/lbtope/">LBtope:</a>&nbsp;A server for predicting linear B-cell epitopes.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/lgepred/">LGEpred:</a>&nbsp;Correlation analysis and prediction of genes expression from amino acid sequence of proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/lpicom/">LPIcom:</a>&nbsp;Analysis and prediction of ligand protein interaction.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/marspred/">MARSpred:&nbsp;</a>Prediction of mitochondrial aminoacyl tRNA synthetases.</li>
<li><a href="http://mpds.osdd.net/">MDPS:</a>&nbsp;A&nbsp;web server&nbsp;designed for&nbsp;in silico drug discovery&nbsp;towards M.&nbsp;<em>tuberculosis</em>.</li>
<li><a href="http://webs.iiitd.edu.in/oscadd/mdri/">MDRIpred</a>: Predicting inhibitor against drug tolrent M.<em>Tuberculosis.</em></li>
<li><a href="http://webs.iiitd.edu.in/raghava/metapred/">MetaPred:&nbsp;</a>Prediction of Cytochrome P450 Isoform responsible for metabolizing a drug molecule.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/mhcbn/">MHCBN:</a>&nbsp;A database of&nbsp;Major Histocompatibility Complex (MHC) Binding, Non-binding peptides .</li>
<li><a href="http://webs.iiitd.edu.in/raghava/tmhcpred/">TmhcPred:</a>&nbsp;A webserver that&nbsp;predict MHC Class I and II binding peptides.</li>
<li><a href="http://crdd.osdd.net/raghava/mhc/" target="_blank">mhc</a>:Matrix Optimization Technique for Predicting MHC binding Core.</li>
<li><a href="http://crdd.osdd.net/raghava/mhcbench/" target="_blank">mhcbench</a>:Evaluation of MHC Binding Peptide Prediction Algorithms.</li>
<li><a href="http://crdd.osdd.net/raghava/mhc2pred/" target="_blank">mhc2pred</a>:SVM based method for prediction of promiscuous MHC class II binders.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/mhcbn/">MHCBN:</a>&nbsp;A database od Major Histocompatibility Complex (MHC) Binding,Non-binding peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/mhcbn/">MHCBN4:</a>&nbsp;A database of MHC/TAP binding peptides and T-cell epitopes.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/mitpred/">MitPred:</a>&nbsp;Prediction of mitochondrial proteins using SVM&nbsp;and hidden Markov model.</li>
<li>&nbsp;<a href="http://webs.iiitd.edu.in/raghava/mtbveb/">MtbVeb:</a>&nbsp;A webportal for designing vaccine against drug resistant and emerging strains of&nbsp;<em>M. tuberculosis</em>.</li>
<li><a href="http://crdd.osdd.net/raghava/mycotb/" target="_blank">MycoTb</a>: A software for managing mycobacterium tuberculosis.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/nadbinder/">NADbinder:</a>&nbsp;Prediction of NAD interacting residues in proteins.</li>
<li><a href="http://crdd.osdd.net/raghava/nagsite/" target="_blank">nagsite</a>:A webserver for the prediction of N-acetyl glucosamine Interacting residue.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/nhlapred/">nHLAPred</a>: A neural network based method for predicting MHC Class-I binders.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/npact/">NPACT:&nbsp;</a>A curated database of plant derived natural compounds that exhibit anti-cancerous activity.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/nppred/" target="new">NPpred:&nbsp;</a>&nbsp;A webserver for the prediction of nuclear proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/nrpred/" target="new">NRpred:&nbsp;</a>&nbsp;A SVM based method for the classification of nuclear receptors&nbsp;.</li>
<li><a href="http://webs.iiitd.edu.in/oscadd/ntegfr/">ntEGFR</a>:&nbsp;QSAR-Based Models for designing inhibitors against Wild and Mutant EGFR (anti-cancer drug).</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ntxpred/" target="new">NTXpred:&nbsp;</a>Identification of neurotoxins their source and function from primary amino acid sequence.</li>
<li><a href="http://webs.iiitd.edu.in/oscadd/">OSCADD:</a>&nbsp;Open Source for Computer-Aided Drug Discovery&nbsp;portal is for chemoinformatics and pharmacoinformatics.</li>
<li><a href="http://crdd.osdd.net/osddchem/">OSDDChem</a>:&nbsp;A interface for large scale synthesis of diverse chemical compounds to screen them against TB and Malaria.</li>
<li><a href="http://webs.iiitd.edu.in/osddlinux/">OSDDlinux</a>: A customize operating system for drug discovery.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/oxdbase/" target="new">OXDbase:&nbsp;</a>A database of Biodegradative oxygenase.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/oxypred/" target="new">OXYpred:&nbsp;</a>Classification and prediction of oxygen binding proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/paaint/" target="_blank">PAAINT</a>: A webservice for predicting residue-residue contacts in proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/parapep" target="_BLANK">ParaPep:&nbsp;</a>Compilation of experimentally validated anti-parasitic peptides and their structures.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pcleavage" target="_BLANK">Pcleavage:&nbsp;</a>Identification of protesosomal cleavage sites in a protein sequence.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pcmdb/" target="new">PCMdb:&nbsp;</a>A database of methylated genes found in pancreatic cancer cell lines and tissues.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/peplife/" target="_BLANK">PEPlife:&nbsp;</a>Database of experimentally validated therapeutic peptides and their half lives.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pepstr/">PEPstr:</a>&nbsp;Prediction of structure of peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pepstrmod/" target="_BLANK">PEPstrMOD</a><a href="http://webs.iiitd.edu.in/raghava/pepstrmod/" target="new">:&nbsp;</a>Structural prediction of peptides containing natural, non-natural and modified residues. &nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pfmpred/" target="new">PFMpred:&nbsp;</a>Predicting mitochondrial proteins of malaria parasite Plasmodium falciparum.</li>
<li><a href="http://www.iiitd.edu.in/raghava/phdcleav/">PHDcleav:</a>&nbsp;Prediction of&nbsp;human Dicer cleavage sites using sequence and secondary structure of miRNA precursors.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pldbench/" target="new">PLDbench:&nbsp;</a>Benchmarking of docking software on peptide-ligand complexes.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/plifepred/" target="new">PlifePred:&nbsp;</a>An insilico method to design and predict the half life of modified and natural peptides in blood.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/polyapred/" target="new">PolyApred:&nbsp;</a>Prediction of polyadenylation signal (PAS) in human DNA sequence.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/polysacdb/" target="new">PolysacDB:&nbsp;</a>&nbsp;A comprehensive Database of microbial Polysaccharide antigens and their antibodies.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/ppdbench/" target="new">PPDBench:&nbsp;</a>&nbsp;Benchmarking of docking software on protein-peptide complexes.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pprint/" target="new">Pprint:&nbsp;</a>&nbsp;A &nbsp;web-server for predicting RNA-binding residues of a protein.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/premier/" target="new">PreMieR:&nbsp;</a>&nbsp;A webserver for the prediction of Mannose Interacting residue.</li>
<li><a href="http://crdd.osdd.net/raghava/proclass/" target="_blank">proclass</a>: protein structure classification server.</li>
<li><a href="http://bioinfo.imtech.res.in/servers/procardb">ProCarDB</a>: A&nbsp;database of bacterial carotenoids.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/proglycprot/" target="new">PROGLYCPROT:&nbsp;</a>&nbsp;Database of&nbsp;experimentally characterized eubacterial and archaeal glycoproteins.</li>
<li>&nbsp;<a href="http://webs.iiitd.edu.in/raghava/propred/">ProPred:</a>&nbsp;A webserver to&nbsp;&nbsp;predict MHC Class-II binding regions in an antigen sequence.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/propred1/">ProPred-1:&nbsp;</a>Prediction of&nbsp;&nbsp;promiscuous MHC Class-I binders.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/proprint/">ProPrInt:</a>&nbsp;A&nbsp;web-server predicts physical or functional interactions between protein molecules.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/prrdb/index.html">PRRDB:</a>&nbsp;A database of pattern recognition receptor.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/psa/">PSAWeb:</a>&nbsp;Analysis of Protein Sequence and Multiple Alignment.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pseapred/">PSEApred:</a>&nbsp;Prediction of Plasmodium Secretory and Infected Erythrocyte Associated Proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/pslpred/">PSLpred:</a>&nbsp;Predict subcellular localization of prokaryotic proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/qaspro/">QASpro:&nbsp;</a>A webserver for the&nbsp;Quality Assesment of Protein Structure.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/rbpred/">RB-Pred:</a>&nbsp;A weather based rice blast prediction server.</li>
<li><a href="http://crdd.osdd.net:8081/RISCbinder/">RISICbinder:</a>&nbsp;Prediction of guide strand of microRNAs from its sequence and secondary structure.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/rnacon/">RNAcon:</a>&nbsp;A&nbsp;web-server for the prediction and classification of non-coding RNAs.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/rnaint/">RNAint:&nbsp;</a>Identification of RNA interacting residues in a protein.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/rnapin/">RNApin:</a>&nbsp;Prediction of Protein Interacting Nucleotides (PINs) in RNA sequences.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/rnapred/">RNApred:</a>&nbsp;A webserver for the prediction of RNA binding proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/rrcpred/">RRCpred:</a>&nbsp;A&nbsp;Webserver for predicting residue-residue contact.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/rslpred/">RSLpred:</a>&nbsp;A method for the subcellular localization prediction of rice proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/sapdb/">SAPdb:</a>&nbsp;A database of nanostructure formed by self assembly of short peptide.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/sarpred/">SARpred:</a>&nbsp;A neural network based method predicts the real value of surface acessibility.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/satpdb/">SATPdb:&nbsp;</a>A&nbsp;database of structurally annotated therapeutic peptides .</li>
<li><a href="http://www.imtech.res.in/raghava/sirnapred/">siRNApred</a>:&nbsp;SVM based method for predicting efficacy value of siRNA.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/srf/">SRF:&nbsp;</a>A program to find&nbsp;repeats through an analysis of the power spectrum of a given DNA sequence.&nbsp;</li>
<li><a href="http://webs.iiitd.edu.in/raghava/srtpred/">SRTpred:</a>&nbsp;A&nbsp;method for the classification of protein sequence as secretory or non-secretory protein.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/starpdb/">STARPDB:</a>&nbsp;A webserver&nbsp;for annotating structure of a protein using simililarity based approach.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/tappred/">TAPPred:&nbsp;</a>A webserver for&nbsp;predicting binding affinity of peptides toward the TAP transporter.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/tbbpred/">TBBpred:</a>&nbsp;A webserver for the prediction of&nbsp;transmembrane Beta barrel regions in a given protein sequence.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/tbpred/">TBpred:</a>&nbsp;A webserver &nbsp;that predicts four subcellular localization&nbsp;of mycobacterial proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/thpdb/">THPdb:</a>&nbsp;A database of FDA approved therapeutic peptides and proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/topicalpdb/index.php">TopicalPdb:</a>&nbsp;A&nbsp;Database of Topically Administered Peptide.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/toxinpred/">ToxinPred:&nbsp;</a>An&nbsp;<em>in silico</em>&nbsp;method, which is developed to predict and design toxic/non-toxic peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/toxipred/">ToxiPred:</a>&nbsp;A server for prediction of aqueous toxicity of small chemical molecules in T. pyriformis.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/trnamod/">tRNAmod:&nbsp;</a>The tRNAmod is a web-server for the prediction of transfer RNA (tRNA) modifications.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/tsppred/">TSP-PRED:&nbsp;</a>A webservice for predicting Tertiary Structure of proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/tumorhope/">TumorHope:</a>&nbsp;A comprehensive database of Tumor Homing Peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/tumorhpd/">TumorHPD:&nbsp;</a>A webserver for predicting and designing of tumor homing peptides.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/vaccineda/">VaccineDA:&nbsp;</a>A webserver for prediction of oligodeoxynucleotide vaccine adjuvant.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/vaxinpad/">VaxinPAD:&nbsp;</a>A webserver for designing peptide based vaccine adjuvant.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/vgichan/">VGIchan:&nbsp;</a>Voltage gated ion channel prediction server.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/vicmpred/">VICMPpred:&nbsp;</a>&nbsp;Prediction of Virulence factors, Information molecule, Cellular process and Metabolism molecule in the Bacterial proteins.</li>
<li><a href="http://webs.iiitd.edu.in/raghava/vitapred/">VitaPred:&nbsp;</a>A webserver for predicting the vitamin interacting&nbsp;residue.</li>
<li><a href="http://webs.iiitd.edu.in/oscadd/xiapin/">XIAPin</a>: Designing of efficient inhibitors against apoptosis protein&nbsp;XIAP (inhibitor against&nbsp;drug resistance cancer).</li>
<li><a href="http://bioinfo.imtech.res.in/manojk/zikavr/">ZikaVR</a>:&nbsp;An Integrated Zika Virus Resource for Genomics, Proteomics, Phylogenetic and Therapeutic Analysis</li>
</ol></div></div></div></div>
